test/StdCheats.t.sol

```diff
@@ -103,6 +103,35 @@ contract StdCheatsTest is Test {
         assertEq(barToken.totalSupply(), 10000e18);
     }
 
+    function testDealERC1155Token() public {
+        BarERC1155 barToken = new BarERC1155();
+        address bar = address(barToken);
+        dealERC1155(bar, address(this), 0, 10000e18, false);
+        assertEq(barToken.balanceOf(address(this), 0), 10000e18);
+    }
+
+    function testDealERC1155TokenAdjustTS() public {
```

Comment:
```suggestion
    function testDealERC1155TokenAdjustTotalSupply() public {
```

Reply:
Should we modify this for the `testDealTokenAdjustTS()` function for the ERC20 deal cheatcode too @mds1 ?
https://github.com/foundry-rs/forge-std/blob/65761f05868d5dcea3e4b076da58c1c3f447940f/test/StdCheats.t.sol#L95

Reply:
Yea that would be ideal IMO, just more clear/explicit that way. Sorry didn't realize that's where you got the name from!

Reply:
Ok, I just pushed this last change